src/tests.js

```diff
@@ -29,54 +30,53 @@ export default class Test {
   }
 
   element(select, callback) {
-    if(!this.helpers) return
+    if(!this.helpers) return this
 
-    let element
     if(typeof select === 'string') {
-      element = this.helpers.elements[select]
-      callback.call(this, element)
+      const elements = this.helpers.elements[select]
+      callback.call(this, elements)
       return this
     }
 
-    element = this.getFirst(this.helpers.elements)
-    select.call(this, element)
+    select.call(this, this.root)
     return this
   }
 
   test(callback) {
-    var params = this.params()
-    callback.call(params, params)
-    return this
-  }
-
-  params(){
-    var length = Object.keys(this.helpers).length
-    if(this.helpers.elements && length === 1) {
-      return Object.assign({}, this, this.helpers.elements)
-    }
+    callback.call(this, this)
     return this
   }
 
   //private
 
   getFirst(object){
-    for (let element in object) return object[element]
+    for (const element in object) return object[element]
   }
 
   //Built in middleware
 
-  find(data){
-    Find.call(this, data)
+  find(...args){
```

Comment:
Why `...args` ?? We are only supposed pass a single variable here. I think it should `find(selector)`.